repos/system_upgrade/common/actors/checkinsightsautoregister/libraries/checkinsightsautoregister.py

```diff
@@ -0,0 +1,53 @@
+from leapp import reporting
+from leapp.libraries.common import rhsm
+from leapp.libraries.common.config import get_env
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRPM, RpmTransactionTasks
+
+INSIGHTS_CLIENT_PKG = "insights-client"
+
+
+def _ensure_package(package):
+    """
+    Produce install tasks if the given package is missing
+
+    :return: True if the install task is poduced else False
```

Comment:
s/poduced/produced

Reply:
Interesting, the codespell action should have caught this I think.